InnowiseSQL.sql

```diff
@@ -1 +1,416 @@
+	   (N'Евгения','Кашняр','Романовна','жен',4),
+	   (N'Екатерина','Мясникова','Алексеевна','жен',1),
+	   (N'Тарас','Красильников','Дмитриевич','муж',5),
+	   (N'Спартак','Белоусов','Дмитриевич','муж',1);
+
+
+
+INSERT INTO Accounts(BankId,Balance, ClientId)
+VALUES (1,1566,1),
+	   (1,0,2),
+	   (1,0,3),
+	   (2,0,4),
+	   (2,78,5),
+	   (2,0,6),
+	   (3,3,7),
+	   (3,29,8),
+	   (3,0,9),
+	   (4,0,10),
+	   (4,19,11),
+	   (4,39,18),
+	   (5,533,22),
+	   (5,309,23),
+	   (5,399,24),
+	   (6,628,19),
+	   (6,570,20),
+	   (6,518,21),
+	   (7,796,25),
+	   (7,96,26),
+	   (7,394,27),
+	   (8,222,28),
+	   (8,638,29),
+	   (8,357,30),
+	   (9,300,12),
+	   (9,1031,13),
+	   (9,5000,14),
+	   (10,1166,15),
+	   (10,200,16),
+	   (10,562,17);
+
+INSERT INTO Cards(Balance,AccountId)
+VALUES (320,1),
+	   (200,1),
+	   (200,1),
+	   (200,1),
+	   (200,1),
+	   (0,4),
+	   (0,4),
+	   (0,4),
+	   (0,4),
+	   (200,24),
+	   (99,24),
+	   (531,13),
+	   (200,27),
+	   (94,27),
+	   (138,29),
+	   (100,23),
+	   (100,23),
+	   (100,23),
+	   (62,17),
+	   (0,10),
+	   (0,10),
+	   (0,14),
+	   (0,14),
+	   (0,7),
+	   (0,7),
+	   (0,7),
+	   (1,7),
+	   (1,7),
+	   (0,3),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (66,15),
+	   (100,15),
+	   (0,28),
+	   (0,9),
+	   (0,9),
+	   (0,16),
+	   (75,20),
+	   (122,22),
+	   (128,19),
+	   (100,19),
+	   (100,19),
+	   (100,19),
+	   (100,19);
+
+
+	--Query 1
+
+SELECT DISTINCT Banks.Name, Towns.Name
+FROM Banks
+INNER JOIN Filials ON Banks.Id = Filials.BankId
+INNER JOIN Buildings ON Filials.Id = Buildings.FilialId
+INNER JOIN Towns ON Buildings.TownId = Towns.Id
+WHERE Towns.Name = 'Минск'
+
+
+	--Query 2
+
+SELECT Banks.Name, Cards.Id, Accounts.Id, Clients.Name, Cards.Balance AS Card_Balance, Accounts.Balance AS Account_Balance
+FROM Banks
+INNER JOIN Accounts ON Accounts.BankId = Banks.Id 
+INNER JOIN Clients ON Accounts.ClientId = Clients.Id
+INNER JOIN Cards ON Cards.AccountId = Accounts.Id;
+
+
+	--Query 3
+
+SELECT Accounts.Id, Accounts.Balance, SUM(Cards.Balance) AS Summa, Accounts.Balance - SUM(Cards.Balance) as dif
+FROM Accounts INNER JOIN Cards ON AccountId = Accounts.Id
+GROUP BY Accounts.Id, Accounts.Balance
+HAVING SUM(Cards.Balance) != Accounts.Balance
+
+	--Query 4a
+
+SELECT SocStatus.Name, COUNT(*) AS CardsAmount
+FROM Clients
+INNER JOIN Accounts ON ClientID = Clients.Id
+INNER JOIN SocStatus ON SocStatus.Id = SocStatusID
+INNER JOIN Cards ON AccountID = Accounts.Id
+GROUP BY SocStatus.Name;
+
+	--Query 4b 
+
+
+	--Query 5
+
+CREATE PROCEDURE AccrualForSocStatus
+@status NVARCHAR
+AS
+BEGIN
+UPDATE Accounts
+SET Balance = Balance + 10
+FROM Accounts
+	INNER JOIN Clients ON Clients.Id = ClientID
+	INNER JOIN SocStatus ON SocStatusId = SocStatus.Id
+WHERE SocStatus.Name = @status
+END
+GO
+
+
+	--Query 6
+
+SELECT AccountID, Accounts.Balance, SUM(Cards.Balance) AS Summa,
+(Accounts.Balance - SUM(Cards.Balance))/COUNT(Cards.Balance) AS AvailableFunds
```

Comment:
![image](https://user-images.githubusercontent.com/33979910/162562632-2471a5c4-9e4f-46f4-893d-e6963da0a752.png)
I think something going wrong here
Check the logic of query